region/maridia/inner-pink.json

```diff
@@ -3680,6 +3805,25 @@
                     }}
                   ]
                 },
+                {
+                  "name": "IBJ",
+                  "notable": false,
+                  "requires": [
+                    "Gravity",
+                    "canPlayInSand",
+                    "canSandIBJ"
```

Comment:
can `canSandIBJ` be bypassed with a springball + jump into ibj? I would think it would be much easier